packages/foreigntypes/posixtypes.pony

```diff
@@ -0,0 +1,95 @@
+// This file defines the standard Posix type to make FFI calls easier
+// to manage and more consistent.
+
+type BlkCnt is I64
+
+// not consistent across OSX (always 32 bit int)/Linux (Long)
+type BlkSize is ILong
+type OSXBlkSize is I32
+
+type Cc is U8
+
+// not consistent across OSX (Unsigned Long)/Linux (Long)
+type Clock is ILong
+type OSXClock is ULong
+
+// not defined on OSX
+type ClockId is I32
```

Comment:
Again, this is a case where I'd rather be able to use `ifdef`s to enforce that it isn't defined on OSX.